src/studio/src/designer/backend.Tests/Controllers/DatamodelsControllerTests.cs

```diff
@@ -289,6 +291,74 @@ public async Task Delete_Datamodel_Ok()
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
+        [Fact]
+        public async Task GetDatamodels_NoInput_ShouldReturnAllModels()
+        {
+            var client = GetTestClient();
```

Comment:
we usually try to avoid using var when the type is known to us.

Reply:
Why? The compiler infers the type from the return type of the calling method. 'var' is just syntactic sugar to avoid typing a type that is already known. I think it should be the other way around, use var when we don't know the returning type or when it's not obvious ref. https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions

Reply:
Yes, this is also an "Altinn 2-ism" we should stop doing.

**Readability** is king, and as long as it's obvious (without intellisense) what type a variable is, we should use `var`.  
Shorter, simpler, easier to read and easier to refactor.

This could also make us create better method names.

Reply:
Also agree that using var when we know the type, makes the code easier to read.
If var is used when the return type is unknown then we might as well write our backend in Node.